modules/internal/ChapelRange.chpl

```diff
@@ -2801,34 +2780,19 @@ private proc isBCPindex(type t) param do
         newAbsStride = newStride;
       }
 
-      if multiplyStrideSigns {
+      {
         // sign of resulting stride = sign of 'this' * sign of 'other'
         if this.hasPositiveStride() && other.hasNegativeStride() ||
            this.hasNegativeStride() && other.hasPositiveStride()
         then
           newStride = -newStride;
-
-      } else {
-        // sign of resulting stride = sign of 'this'
-        if this.hasNegativeStride() then
-          newStride = - newStride;
```

Comment:
Why is this block needed? Can it just sit by itself?

Reply:
The block is not needed. I made it simply to reduce the amount of (whitespace) changes.

Reply:
To follow up, I removed the unneeded curlies.